packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx

```diff
@@ -24,4 +32,42 @@ export class ViewerNavigationToolsProvider extends StandardNavigationToolsProvid
       },
     });
   }
+
+  public override provideToolbarButtonItems(
+    stageId: string,
+    stageUsage: string,
+    toolbarUsage: ToolbarUsage,
+    toolbarOrientation: ToolbarOrientation
+  ): CommonToolbarItem[] {
+    const items: CommonToolbarItem[] = super.provideToolbarButtonItems(
+      stageId,
+      stageUsage,
+      toolbarUsage,
+      toolbarOrientation
+    );
+
+    if (
+      stageUsage === StageUsage.General &&
+      toolbarUsage === ToolbarUsage.ViewNavigation &&
+      toolbarOrientation === ToolbarOrientation.Vertical
+    ) {
+      items.push(
+        ToolbarHelper.createToolbarItemFromItemDef(
+          10,
+          new ToolItemDef({
+            toolId: "View.LookAndMove",
+            iconSpec: "icon-walk",
+            execute: () =>
+              IModelApp.tools.run(
+                "View.LookAndMove",
```

Comment:
did you see if there was a Command for this?

Reply:
Is [this](https://github.com/iTwin/itwinjs-core/blob/112933a965e339f36b8af4f092fae8c73f6767da/core/frontend/src/tools/Tool.ts#L865) the command you mean?

Reply:
Wondering if there is a command for this tool like the old one, see https://github.com/iTwin/itwinjs-core/blob/master/ui/appui-react/src/appui-react/ui-items-provider/StandardNavigationToolsProvider.ts#L64

Reply:
ideally wed use ToolbarHelper.createToolbarItemFromItemDef, and just pass in the command if one exists

Reply:
You mean one of the commands from [CoreTools](https://github.com/iTwin/itwinjs-core/blob/007be59f93813bc5525b1fe2afc48f20761bca56/common/api/appui-react.api.md?plain=1#L1413), or somewhere else?

Reply:
yea i was wondering if the there was a command (or i guess an ItemDef) for it somewhere in the repo.

looks like its the LookAndMoveTool, but there is no itemDef command for it, so i guess this is fine